eng/tools/mgmtreport/codecoverage.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v6/test"
+)
+
+func getBuildCodeCoverage(client *azuredevops.Client, projectName string, buildId int) (*test.CodeCoverageSummary, error) {
+	request, err := client.CreateRequestMessage(
```

Comment:
Since `azuredevops` module has some issue of fetching code coverage (refer issue), we need to do that by ourselves.